src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  function ButtonDiv() {
+    return (
+      <div className="parent">
+        <button className="button">AC</button>
+        <button className="button">+/-</button>
+        <button className="button">%</button>
+        <button className="button-color">/</button>
+        <button className="button">7</button>
+        <button className="button">8</button>
+        <button className="button">9</button>
+        <button className="button-color">X</button>
+        <button className="button">4</button>
+        <button className="button">5</button>
+        <button className="button">6</button>
+        <button className="button-color">-</button>
+        <button className="button">1</button>
+        <button className="button">2</button>
+        <button className="button">3</button>
+        <button className="button-color">+</button>
+        <button className="button-zero">0</button>
+        <button className="button">.</button>
+        <button className="button-color">=</button>
+      </div>
+    );
+  }
```

Comment:
- [ ] Nice work creating the `Calculator.js` component :clap: However it would be nice to move this `ButtonDiv` function to a new component (`ButtonDiv.js`) and import it in the `Calculator.js`. That will make your code clean and also make you follow the project requirement to have at least **1 child component**
```js
import ButtonDiv from './ButtonDiv';
```